pkg/controller/statefulset/stateful_set_control.go

```diff
@@ -110,19 +110,15 @@ func (ssc *defaultStatefulSetControl) performUpdate(
 	if err != nil {
 		return currentRevision, updateRevision, currentStatus, err
 	}
-	klog.V(4).Infof("StatefulSet %s/%s pod status replicas=%d ready=%d current=%d updated=%d",
-		set.Namespace,
-		set.Name,
-		currentStatus.Replicas,
-		currentStatus.ReadyReplicas,
-		currentStatus.CurrentReplicas,
-		currentStatus.UpdatedReplicas)
-
-	klog.V(4).Infof("StatefulSet %s/%s revisions current=%s update=%s",
-		set.Namespace,
-		set.Name,
-		currentStatus.CurrentRevision,
-		currentStatus.UpdateRevision)
+	klog.V(4).InfoS("StatefulSet pod status", "statefulset", klog.KObj(set),
+		"replicas", currentStatus.Replicas,
```

Comment:
```suggestion
		"replicaSet", currentStatus.Replicas,
```

Reply:
'replicas' means replica number, not replicaSet

Reply:
yeah, You are right, sorry.